src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -292,6 +292,8 @@ public partial class Socket : System.IDisposable
         [System.ObsoleteAttribute("SupportsIPv6 is obsoleted for this type, please use OSSupportsIPv6 instead. https://go.microsoft.com/fwlink/?linkid=14202")]
         public static bool SupportsIPv6 { get { throw null; } }
         public short Ttl { get { throw null; } set { } }
+        [System.ObsoleteAttribute("This property has no effect in .NET 5+ and .NET Core.")]
```

Comment:
@terrajobst, we really need a better way to refer to this :)  For anything that started off in .NET Core, are we now going to have to say .NET Core and .NET 5+?

Reply:
The [official guide](https://docs.microsoft.com/en-us/contribute/dotnet/net-core-5-naming) suggests to refer to it as 

> ".NET" (without mention of 5 or Core)

which would be terribly confusing in this case. ("This property has no effect in .NET" 🤷)

Reply:
Right,  but so is ".NET 5+ and .NET Core", as it incorrectly implies that .NET 5 isn't .NET Core.

cc: @richlander

Reply:
> ".NET" (without mention of 5 or Core)

When you need to emphasize specific implementation (the case here), the guide suggests to use ".NET 5+ (and .NET Core)" for first reference and ".NET 5+" for subsequent references.

This guide was created just recently. I did not know about that. @antonfirsov thank you for sharing the link. There is a ton of places everywhere that use the phrase like the one used by this PR that was de-facto standard befofe.

Reply:
> the guide suggests to use ".NET 5+ (and .NET Core)" 

Actually, it suggests:

> .NET 5 (and .NET Core) and later versions

which is terribly verbose (at a level that makes it obscure IMO), this is why I went with `.NET 5+ and .NET Core`. I believe we should revisit that guide.

> Right, but so is ".NET 5+ and .NET Core", as it incorrectly implies that .NET 5 isn't .NET Core.

@stephentoub although from internal (dotnet/runtime developer) perspective `.NET 5 ⊂ .NET Core` (being just another release), from what I see from the guide, this is not what's being communicated on the branding front.

Maybe we should open a tracking issue, to revisit the terminology? If yes where?

Reply:
> .NET 5 (and .NET Core) and later versions

"and later versions" is the standardized docs terminology. Yes, it is verbose and it is often abbreviated. Perhaps the guide should mention the `.NET 5+` abbreviation as an option. For example, this abbrevation is often used in https://github.com/dotnet/dotnet-api-docs currently.

> Maybe we should open a tracking issue, to revisit the terminology? If yes where?

The docs repo. The document that describes the terminology is there.